cli/pyproject.toml

```diff
@@ -1,13 +1,13 @@
 [project]
 name = "pixl_cli"
-version = "0.1.0"
+version = "0.2.0-rc"
 authors = [{ name = "PIXL authors" }]
 description = "PIXL command line interface"
 readme = "README.md"
 requires-python = ">=3.10"
 classifiers = ["Programming Language :: Python :: 3"]
 dependencies = [
-    "core==0.1.0",
+    "core>=0.1.0",
```

Comment:
should this be pinned rather than a minimum version?

Reply:
Yeah I'd probably pin it to the rc